app/models/task.py

```diff
@@ -1,6 +1,25 @@
-from flask import current_app
+from flask import current_app, jsonify
 from app import db
-
+import requests
+import os
+#from app.models.goal import Goal
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable =True)
+    is_complete = db.Column(db.Boolean)
```

Comment:
`is_complete` should not be a separate attribute. `is_complete` us False when `completed_at` is None and `True` when `completed_at` has a value as you calculated below on line 20 (nice work there!). The danger in adding `is_complete` as another attribute on the task model is that we could store data that is inconsistent